cmd/root.go

```diff
@@ -100,9 +113,24 @@ func flagsLogCollector() {
 
 // Initialize the cobra command CLI.
 func init() {
+
+	// All global flag
+	rootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false,"Enable verbose or debug logging")
+	rootCmd.PersistentFlags().BoolVarP(&logfile, "log-file", "l", false, "Enable recording all the log messages to the logfile")
+	rootCmd.PersistentFlags().StringVarP(&logDestination, "log-destination", "d", "/tmp", "Directory where the logfile should be created, only works with --log-file flag")
+
+	// Database connection parameters.
+	rootCmd.PersistentFlags().StringVarP(&hostname, "hostname", "", "localhost","Hostname where the database is hosted")
+	rootCmd.PersistentFlags().IntVarP(&port, "port", "", 5432, "Port number of the master database")
+	rootCmd.PersistentFlags().StringVarP(&database, "database", "", "template1", "Database name to connect")
```

Comment:
These can be just `StringVar` instead of `StringVarP` if we don't allow a short option.